mne/viz/tests/test_topomap.py

```diff
@@ -86,6 +86,7 @@ def test_plot_topomap():
     evoked.plot_topomap(times, ch_type='planar2', res=res)
     evoked.plot_topomap(times, ch_type='grad', mask=mask, res=res,
                         show_names=True, mask_params={'marker': 'x'})
+    evoked.plot_topomap("peaks")
```

Comment:
I'm not sure how that could work - plot_topomap only returns the times it plots at as titles for the subplots right? So maybe

    f = evoked.plot_topomap("peaks")
    peaks = [float(str(ax.title).split("'")[1].split()[0]) 
             for ax in f.axes[:-1]]
    mx = evoked.times[evoked.data.std(axis=0).argmax()]
    assert_true(any(np.isclose(mx * 1000, peaks, atol=1)))